deploy.tf

```diff
@@ -3,6 +3,10 @@ provider "azurerm" {
   features {}
 }
 
+locals {
+  sp_json = jsondecode(file("${path.module}/azureauth.json"))
```

Comment:
You could pass this in using TF_VAR: https://www.terraform.io/docs/configuration/variables.html#environment-variables

Reply:
true, tho [create-service-principal.sh](./create-service-principal.sh) is used to write creds to a specific local file, so i think this is okay... ?